src/hooks/useVideoSearch.ts

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+import youtube from "../apis/youtube";
+import { Video } from "../types/types";
+
+export default function useVideoSearch(searchTerm: string) {
+  const [currentTerm, setCurrentTerm] = useState<string>("");
+  const [data, setData] = useState<Video[]>([]);
+  const [nextPageToken, setNextPageToken] = useState<string>("");
+  const [prevPageToken, setPrevPageToken] = useState<string>("");
```

Comment:
Line 9:10:  'prevPageToken' is assigned a value but never used  @typescript-eslint/no-unused-vars